packages/files/file.pony

```diff
@@ -1,4 +1,21 @@
-primitive _FileHandle
+use @_read[I32](fd: I32, buffer: Pointer[None], bytes_to_read: I32) if windows
+use @read[ISize](fd: I32, buffer: Pointer[None], bytes_to_read: USize) if not windows
+use @_write[I32](fd: I32, buffer: Pointer[None], bytes_to_send: I32) if windows
+use @writev[ISize](fd: I32, buffer: Pointer[None], num_to_send: I32) if not windows
+use @_lseeki64[I64](fd: I32, offset: I64, base: I32) if windows
+use @lseek[I64](fd: I32, offset: I64, base: I32) if not windows
```

Comment:
Ah, now I understand your concern. One test I can think of is a compile (cross compile) a 32 bit version of a Pony program that checks the size of a file and confirm that the program shows the correct size.

Does that sound like a sufficient test to confirm it works correctly (uses 64 bit functions) on a 32 bit platform?